mongofiles/options.go

```diff
@@ -28,6 +28,7 @@ Possible commands include:
 	put_id    - add a file with filename 'filename' and a given '_id'
 	get       - get files with filenames specified in the supporting arguments
 	get_id    - get a file with the given '_id'
+        get_regex - get files matching the supplied 'regex'
```

Comment:
```suggestion
	get_regex - get files matching the supplied 'regex'
```
[nit] tabs vs. spaces

Reply:
Shouldn't this be handled by `gofmt` and # verified by the `format-go` and `lint-go` Evergreen tasks?

Reply:
No, it wouldn't. This indentation is part of a string variable, and so wouldn't be changed (or even inspected) `by gofmt`